Content.Server/Implants/SubdermalImplantSystem.cs

```diff
@@ -25,6 +25,7 @@
 using Content.Shared.Store.Components;
 using Robust.Shared.Collections;
 using Robust.Shared.Map.Components;
+using Content.Server.DetailExaminable;
```

Comment:
```suggestion
using Content.Server.DetailExaminable;
using Content.Shared.Store.Components;
using Robust.Shared.Collections;
using Robust.Shared.Map.Components;
```
lets keep the imports sorted alphabetically